setup.py

```diff
@@ -430,4 +430,14 @@ def has_ext_modules():
     # As we do version specific hacks for installed inline copies, make the
     # wheel version and platform specific.
     distclass=BinaryDistribution,
+    extras_require={
+        "all": [
+            "ordered-set; python_version >= '3.10'",
+            "subprocess32; python_version == '2.7'",
+            "zstandard; python_version >= '3.5'",
+        ],
+        "msvc": [
+            "orderedset; python_version >= '3.6' or os.name != 'nt'",
```

Comment:
For orderedset, it's way more of a mess. Should be more like this:

```
orderedset == 2.0.3 ; os.name != "nt" and sys.platform != "darwin" and python_version < "3.7"
orderedset == 2.0.3 ; os.name == "nt" and python_version >= '3.4' and python_version < "3.7"
orderedset == 2.0.3 ; sys.platform == "darwin" and python_version < "3.7"
ordered-set == 4.1.0; python_version >= "3.7"
```

and it's not for MSVC, but I think I get the idea.

Reply:
Sure! (I just copy-and-pasted blindy from `requirements.txt`.)